be/src/util/bitmap_value.h

```diff
@@ -1259,10 +1259,9 @@ class BitmapValue {
     }
 
     static std::string empty_bitmap() {
-        static BitmapValue bitmap;
-        std::string buf;
-        buf.resize(bitmap.getSizeInBytes());
-        bitmap.write_to(buf.data());
+        std::string buf(sizeof(BitmapValue), 0);
```

Comment:
why the original code will core???

Reply:
BitmapValue::write_to  will get a string with size 1 for empty BitmapValue,  however the size 1 string will reinterpret to BitmapValue* back in ColumnComplexType::insert:
 void insert(const Field& x) override {
        const String& s = doris::vectorized::get<const String&>(x);
        data.push_back(*reinterpret_cast<const T*>(s.c_str()));
    }

in data.push_back will goto BitmapValue copy constructor,  as the _type is not first member in BitmapValue, cause  access to an unknown memory location.